protocols/gossipsub/src/peer_score.rs

```diff
@@ -780,6 +780,11 @@ impl PeerScore {
         }
     }
 
+    /// Returns a scoring parameters for a topic if existent.
+    pub(crate) fn get_topic_params(&self, topic_hash: &TopicHash) -> Option<TopicScoreParams> {
```

Comment:
should be enough to return a reference. If the caller needs a value they can clone it